frontend/src/layouts/DashboardLayout_gdd.vue

```diff
@@ -45,6 +45,7 @@ import { FadeTransition } from 'vue2-transitions'
 import CONFIG from '@/config'
 
 export default {
+  name: 'DashboardTemplate',
```

Comment:
```suggestion
  name: 'DashboardLayout',
```
Also consider to change the file name to `DashboardLayout.vue`

Reply:
changed